cms/publications/templates/publications/toc.html

```diff
@@ -1,6 +1,6 @@
 {% if toc_items %}
   <div>
-    <h2>Contents</h2>
+    <h2>Content</h2>
```

Comment:
Even more bikesheddy: I'd say Contents was right as a header for the table of contents (as opposed to the technical term for 'this piece of text and images', where 'the content' makes sense)